src/Altinn.Platform/Altinn.Platform.Events/Tests/Data/Profile/User/1.json

```diff
@@ -0,0 +1,14 @@
+{
+  "UserId": 1,
+  "UserName": "OlaNordmann",
+  "PhoneNumber": "12345678",
+  "Email": "test@test.com",
+  "PartyId": 1,
```

Comment:
Lets avoid having overlapping user and partyIds so we can catch potential bugs. 
Same goes for all of these test user's I guess.

Reply:
Promise to fix this in the next pull request.